### PR TITLE
Completed html-web-page exercise - Julie Dumbrique

### PR DESCRIPTION
**Pull request by juliedumbrique**
https://user-images.githubusercontent.com/72038937/129834740-98d016f9-beba-4a25-8440-672797eb0140.mov

### CODE REVIEW

File: html-web-page/index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      <tr>
+        <td>Monthly Pricing</td>
+        <td>$30</td>
+        <td>$50</td>
+        <td>$60</td>
+      </tr>
+    </tbody>
+  </table>
+  <p>Like what you see? Become a part of the JOTMC family today!</p>
+  <label for="name"></label>
```
Comment (codingcodymiller) [inline html-web-page/index.html]: There is no text labelling this input, so no `label` element is necessary

File: html-web-page/index.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  <input type="text" id="name" name="name" value="Your Name">
+  <select name="plan" id="plan-select">
+    <option value="">Select your plan:</option>
+    <option value="basic">Basic Plan</option>
+    <option value="deluxe">Deluxe Plan</option>
+    <option value="giant">Jelly Giant Plan</option>
+   </select>
+  <input type="submit" value="Submit">
```
Comment (codingcodymiller) [inline html-web-page/index.html]: Form control elements should always be children of a `form` element.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start here! A couple notes for you:

**Comment by juliedumbrique** [comment]
Screen Shot 2021-08-18 at 11 05 18 PM

**Review by codingcodymiller — APPROVED** [review]
Great work!!

**Pull request merged**